DuggaSys/microservices/sectionedService/setVisibleListentries_ms.php

```diff
@@ -24,11 +24,13 @@
 if (checklogin() && isSuperUser(getUid())){
 
     // prepare SQL query
-    $query = $pdo->prepare("UPDATE listentries Set visibility = :listentryId");
+    $query = $pdo->prepare("UPDATE listentries SET visibility = :visibility WHERE listentryId = :listentryId");
```

Comment:
the columns are not correctly named
![image](https://github.com/HGustavs/LenaSYS/assets/128985931/3a7c9ec6-f862-4084-b78f-0fdaf9085a28)

I presume it is the `lid` and `visible` columns that should be used here